backend-api/src/functions/asyncTxmaEvent/asyncTxmaEventHandler.ts

```diff
@@ -102,7 +102,11 @@ async function handleGetSessionError({
 }: {
   errorData: GetSessionFailed;
 }): Promise<APIGatewayProxyResult> {
-  if (errorData.errorType === GetSessionError.CLIENT_ERROR) {
+  const { errorType } = errorData;
```

Comment:
```suggestion
  if ([GetSessionError.SESSION_NOT_FOUND, GetSessionError.SESSION_NOT_VALID].includes(errorData.errorType)) {
```

^ very much a nitpick, feel free to ignore if you think the original is more readable